test/runtests.jl

```diff
@@ -2,3 +2,10 @@ using Test, Example
 
 @test hello("Julia") == "Hello, Julia"
 @test domath(2.0) ≈ 7.0
+
+@testset "_dbconnect function for SQLite" begin
+
+    conn= _dbconnect(SQLite.DB, "DBConnector.jl/test/data/database.db")
+    @test @isdefined conn
```

Comment:
I think this is a good test but could you also write a test that:

1. Queries the database
2. Reports a result
3. Verifies that the result returns what is expected